src/celeritas/mat/MaterialView.hh

```diff
@@ -267,6 +270,15 @@ CELER_FUNCTION real_type MaterialView::radiation_length() const
     return this->material_def().rad_length;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Inverse effective atomic mass to the 2/3 power [1/mass^2/3].
```

Comment:
Is this quantity something that's going to be used across models? If not then it could go in the okwenzel params.

Reply:
I don't think it will be used by other models. I had it in okwvi originally and then moved it to the material params for some reason I can't recall 🤷 but yeah I think it makes more sense to have it there.